website/docs/cli/commands/output.mdx

```diff
@@ -85,12 +74,7 @@ $ terraform output password
 password = notasecurepassword
 ```
 
-However, Terraform does not print out `ephemeral` values even if you specify an output by name. Ephemeral values are never stored in state or included in Terraform plans.
-
-```shellsession
-$ terraform output session_token
-session_token = <ephemeral>
-```
+However, Terraform completely omits any `ephemeral` values, even if you specify an output by name. Ephemeral values are never stored in state or included in Terraform plans. For more information, refer to [Sensitive data in state](/terraform/language/state/sensitive-data).
```

Comment:
I think the first sentence could be a little misleading here. The reason is that `terraform output` is always only concerned about root outputs consumable by humans or programs external to Terraform. It never allowed any interaction with child module outputs. So, we are not "omitting" it here, it just isn't allowed to exist.

I'm not sure what the best way of explaining it is but I found the previous sentence above a bit clearer.

> Ephemeral outputs are only available in child modules, you cannot mark an output as `ephemeral` in the root module.